src/libecalc/application/graph_result.py

```diff
@@ -1249,11 +1249,18 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
 
             sub_components.append(obj)
 
-        regularity_emitters = TimeSeriesFloat(
-            values=[1] * self.variables_map.length,
-            unit=Unit.NONE,
-            timesteps=self.variables_map.time_vector,
-        )
+        # When only venting emitters are specified, without a generator set: the installation result is empty.
+        # Ensure that the installation regularity is found, even if only venting emitters are defined:
+        regularity_emitters = {
```

Comment:
It shouldn't be necessary with a specific regularity for emitters, since it's all the same within an installation. But I guess for the others we use the evaluated regularity from installation result?

Reply:
@jsolaas : The challenge is that  installation result is empty, if we have only venting emitters (without fuel consumers or gensets). So it is basically the installation regularity that is read. Yes, it is the evaluated regularity from installation result that is used. Suggestions for other workaround here?

Reply:
`regularity_emitters` is renamed to `regularities`, and is now only set if installation result don´t exist. Ok?